buildinfo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "1.1.26": {
+    "sha1": "5736da3d068ee74b282926b99ad0255368569b51",
+    "tags": ["1.1.26", "1.1", "latest"]
+  },
```

Comment:
We are missing the stable tag here.

Reply:
Yeah, I didn't want to put the whole buildinfo in manually as the (yet-to-be-updated) watchdog should generate this file